### PR TITLE
Translate Groups.md

### PR DESCRIPTION
**Pull request by ecwu**
**Deadly Typo in the Branch Name**

### CODE REVIEW

File: Groups.md [hunk 1]
```diff
@@ -1,99 +1,102 @@
...
+# UIC Homebrew Computer Club Groups Planning Plans
 
-_2017-8-31：本规则尚未完成。_
+| Date      | Changelog                             |
+| --------- | ------------------------------------- |
+| 2017-8-31 | This plan has not yet been completed. |
+| 2017-9-01 | Translating the Document to en-US.    |
 
-UIC Homebrew 计算机俱乐部（缩写 UICHCC）（目前）是一个非社团兴趣组织。其运作方式与 TEDxUIC 形式类似，即俱乐部内部人员负责维护俱乐部运转及参与项目开发，而**一切产出内容面向社会开放**。
+UIC Homebrew Computer Club (Abbreviation: UICHCC) is an interest-driven organization. Its mode of operation is similar to TEDxUIC. a.k.a. The club's internal member responsible for maintaining the operation of the club and participate in project development, and all the creation produce within the UICHCC will be open to the Public.
```
Comment (jyhi) [inline Groups.md]: "a.k.a." = "also known as", used when a noun has a more common alias. For here I think "That is, ..." will be better.

File: Groups.md [hunk 2]
```diff
@@ -1,99 +1,102 @@
...
 
 聚集 Linux 操作系统用户，进行一系列关于 Linux 的实践与分享活动。
 
-### 本地化工作小组
+### l10n Group
 
 对开源软件界面语言及文档、科技文章等进行本地化（翻译、汉化）。计划与外部社区合作，如 LCTT、AOSC 等。
 
-### 开源与自由软件及知识共享促进小组
+### Open source , freeware and Creative Commons Popularize Group
```
Comment (jyhi) [inline Groups.md]: ~~freeware~~ **Free Software**.

File: Groups.md [hunk 3]
```diff
@@ -1,99 +1,102 @@
-# UIC Homebrew Computer Club 小组规划（总）
+# UIC Homebrew Computer Club Groups Planning Plans
```
Comment (jyhi) [inline Groups.md]: A single "plan" should be enough.

File: Groups.md [hunk 4]
```diff
@@ -1,99 +1,102 @@
...
 
-_2017-8-31：本规则尚未完成。_
+| Date      | Changelog                             |
+| --------- | ------------------------------------- |
+| 2017-8-31 | This plan has not yet been completed. |
+| 2017-9-01 | Translating the Document to en-US.    |
```
Comment (jyhi) [inline Groups.md]: Hmm, nice, but a list behind the whole document could be(look) better.

**Review by jyhi — CHANGES_REQUESTED** [review]
Generally a good translation, especially "101"(?) and "R&D". Some use of words need to be further enhanced and there are still fields not translated. Please go on.

File: Groups.md [hunk 5]
```diff
@@ -1,99 +1,102 @@
...
+**TBD: ** To be decided
 
-### 黑客马拉松 (Hackathon) 活动开发组
+### Hackathon
 
-介绍与职务暂定。
+**TBD: ** To be decided
 
-## 项目开发
+## R&D
```
Comment (ecwu) [inline Groups.md]: @lmy441900 Is it appropriate to name "项目开发" to "R&D(Research & Development) Category"?

**Comment by ecwu** [comment]
@ecwu 因为这个（看描述）

File: Groups.md (hunk 5 above)
Comment (jyhi) [reply Groups.md]: No need to change (

File: Groups.md [hunk 6]
```diff
@@ -35,7 +31,7 @@ UIC Homebrew Computer Club (Abbreviation: UICHCC) is an interest-driven organiza
 
 负责进行各类科技内容的发掘，一般是撰写推送文章、产品评测、IT 事件社评。
 
-## PR & Propaganda Group
+## PR(Public Relationship) & Propaganda Group
```
Comment (jyhi) [inline Groups.md]: Space.

```
PR (Public Relationship)
  ^
```

File: Groups.md [hunk 7]
```diff
@@ -49,7 +45,7 @@ UIC Homebrew Computer Club (Abbreviation: UICHCC) is an interest-driven organiza
 
 **TBD: ** To be decided
 
-### Cyber Security & Self-Protection 101
+### Cyber Security & Self-Protection Course
```
Comment (jyhi) [inline Groups.md]: I said 101 was okay (although I have no idea what is it)

**Review by jyhi — CHANGES_REQUESTED** [review]
Good. Please finish the rest.

File: Groups.md (hunk 7 above)
Comment (ecwu) [reply Groups.md]: > In American university course numbering systems, the number 101 is often used for an introductory course at a beginner's level in a department's subject area. This common numbering system was designed to make transfer between colleges easier. In theory, any numbered course in one academic institution should bring a student to the same standard as a similarly numbered course at other institutions. The term was first introduced by the University of Buffalo in 1929.
>
> Based on this usage, the term "101" has been extended to mean an introductory level of learning or a collection of introductory materials to a topic.

[101 (number): In education @ Wikipedia](https://en.wikipedia.org/wiki/101_(number)#In_education)

File: Groups.md [hunk 8]
```diff
@@ -1,99 +1,116 @@
-# UIC Homebrew Computer Club 小组规划（总）
+# UIC Homebrew Computer Club Groups Plan
 
-_2017-8-31：本规则尚未完成。_
 
-UIC Homebrew 计算机俱乐部（缩写 UICHCC）（目前）是一个非社团兴趣组织。其运作方式与 TEDxUIC 形式类似，即俱乐部内部人员负责维护俱乐部运转及参与项目开发，而**一切产出内容面向社会开放**。
+UIC Homebrew Computer Club (Abbreviation: UICHCC) is an interest-driven organization. Its mode of operation is similar to TEDxUIC. That is, the club's internal member responsible for maintaining the operation of the club and participate in project development, and all the creation produce within the UICHCC will be open to the Public.
 
 俱乐部采用（较为松散的）星状拓扑组织方式，即设立一个中心小组（主根组），其下设立项目小组（理论上可以存在无限多个），统一由俱乐部主根组管理。不同于“部门”的概念，这些小组可随意增设、撤销而不影响俱乐部本身运行。工作于某个小组的成员不限制其活动范围，即小组的设立仅用于管理与宣传，**任何人可以任意参与俱乐部所有的项目组（项目组另有规定除外，俱乐部基础设施与服务除外）**。
```
Comment (ecwu) [inline Groups.md]: I will translate this part later.

File: Groups.md [hunk 9]
```diff
@@ -1,99 +1,115 @@
...
 
-### UIC Portal APP 开发小组
+Note: The Open Source Community has a higher independence within UICHCC. But It still belongs to the UICHCC.
 
-介绍与职务暂定。
+- It has its own public relationship.
+- Decide its own affair without the consent of the `\` Group
 
-## 开源社区工作小组
+The community and UICHCC should be advertised at the same.
```
Comment (jyhi) [inline Groups.md:96]: at the same **time**.

**Review by jyhi — APPROVED** [review]
Anyways.

**Pull request merged**